src/sql/sql.go

```diff
@@ -0,0 +1,22 @@
+package sql
+
+import (
+	"github.com/umisama/go-regexpcache"
+	"strings"
+)
+
+// Taken from UI: https://github.com/keboola/kbc-ui/blob/master/src/scripts/modules/transformations/utils/splitSqlQueriesWorker.js
+// Taken and modified from: http://stackoverflow.com/questions/4747808/split-mysql-queries-in-array-each-queries-separated-by/5610067#5610067
+const splitSqlRegexp = `\s*((?:'[^'\\]*(?:\\.[^'\\]*)*'|"[^"\\]*(?:\\.[^"\\]*)*"|\/\*[^*]*\*+(?:[^*/][^*]*\*+)*\/|#.*|--.*|[^"';#])+(?:;|$))`
```

Comment:
Regularny vyraz som zobral od UIcka, musi to fungovat rovnako:
https://keboola.slack.com/archives/CMTBFNLF8/p1625669115049400

Ak sa v UIcku edituje nejaka transformacia, tak sa to zobrazuje ako jedno SQLko, ale v konfiguracii je to pole statements.